CONTINUOUS_INTEGRATION.md

```diff
@@ -0,0 +1,198 @@
+
+An example of a _Job_ within a `DeepSpeech` _Workflow_ (the `build-and-test.yml` _Workflow_) is shown below. You can see it has several _Steps_, some of which are commands, and some of which are _Actions_.
+
+```
+jobs:
+  # Linux jobs
+  swig_Windows_crosscompiled:
+    name: "Lin|Build SWIG for Windows"
+    runs-on: ubuntu-20.04
+    env:
+      CFLAGS: "-static-libgcc -static-libstdc++"
+      CXXFLAGS: "-static-libgcc -static-libstdc++"
+    steps:
+      - run: |
+          sudo apt-get install -y --no-install-recommends autoconf automake bison build-essential mingw-w64
+      - uses: actions/checkout@v2
+        with:
+          repository: "lissyx/swig"
+          ref: "fec7d5d3179833e37759ffc6532f86344982e26a"
+      - run: |
+          mkdir -p build-static/
+      - run: |
+          curl -sSL https://ftp.pcre.org/pub/pcre/pcre-8.43.tar.gz > pcre-8.43.tar.gz
+          ./Tools/pcre-build.sh --host=x86_64-w64-mingw32
+      - run: |
+          sh autogen.sh
+          ./configure --host=x86_64-w64-mingw32 \
+            --prefix=`pwd`/build-static/ \
+            --program-prefix=ds-
+      - run: |
+          make -j
+      - run: |
+          make install
+      - uses: actions/upload-artifact@v2
+        with:
+          name: ${{ github.job }}
+          path: ${{ github.workspace }}/build-static/
+```
+
+The _Actions_ specific for `DeepSpeech` are housed in the `.github/actions` directory of the repository:
+
+```
+~/DeepSpeech/.github/actions$ ls
+total 64
+4 drwxrwxr-x 16 root root 4096 Apr 20 22:52 ./
+4 drwxrwxr-x  4 root root 4096 Apr 20 22:52 ../
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 build-tensorflow/
+4 drwxrwxr-x  3 root root 4096 Apr 20 22:52 check_artifact_exists/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 get_cache_key/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 host-build/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 install-python-upstream/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 node-build/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 numpy_vers/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 package/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 package-tensorflow/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 python-build/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 run-tests/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 select-xcode/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 setup-tensorflow/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 win-install-sox/
+```
+
+Within each of these directories is a `.yml` file that defines an _Action_. Each _Action_ calls a `bash` script. The `bash` scripts are held in the `ci_scripts` directory in the `DeepSpeech` repository:
```

Comment:
That's not completely true, some actions are written in NodeJS (e.g., `check_artifact_exists`) because it is easier to interact with the GitHub API this way.

Reply:
Thanks Alex, I have amended this wording to reflect your feedback.